src/target/riscv/riscv-013.c

```diff
@@ -4459,8 +4424,10 @@ riscv_insn_t riscv013_read_debug_buffer(struct target *target, unsigned index)
 int riscv013_invalidate_cached_debug_buffer(struct target *target)
 {
 	dm013_info_t *dm = get_dm(target);
-	if (!dm)
+	if (!dm) {
+		LOG_TARGET_ERROR(target, "No DM is specified for the target");
```

Comment:
Have you please come across a scenario when this error message would pop up?

Since there is no progbuf_cache to invalidate if the target does not have a debug module specified, I'd recommend to demote this to a debug message only.

Reply:
Thank you for the suggestion! I've added the message just in case. Have not observed it yet. I've demoted the message as you recommended.